cookbooks/smoke-tests/templates/default/hive_smoke_test.hql.erb

```diff
@@ -0,0 +1,9 @@
+CREATE DATABASE ${HIVE_SMOKE_TEST_DB};
+USE ${HIVE_SMOKE_TEST_DB};
+CREATE TABLE IF NOT EXISTS ${HIVE_SMOKE_TEST_TBL}(firstName string, lastName string);
+INSERT INTO TABLE ${HIVE_SMOKE_TEST_TBL} VALUES ('firstName1', 'lastName');
+
+SELECT firstName FROM ${HIVE_SMOKE_TEST_TBL};
+
+DROP TABLE ${HIVE_SMOKE_TEST_TBL};
```

Comment:
Should this be a prepare step to ensure all's clean at the start of the run incase the last test failed before cleaning up?

Reply:
The intent is that script should only drop what it creates. As we have intentions to run these tests in productions, I would avoid to blind delete/drop statements.

Reply:
While I certainly advocate not wiping all databases, the drop on line 8 seems like it'd be better to move to line 1. That way we can investigate the test results after running and the test should not be influenced by the last test, if it were to have failed.